mailtrap.gemspec

```diff
@@ -19,6 +19,9 @@ Gem::Specification.new do |spec|
   spec.metadata['changelog_uri'] = 'https://github.com/railsware/mailtrap-ruby/blob/main/CHANGELOG.md'
   spec.metadata['rubygems_mfa_required'] = 'true'
 
+  spec.add_dependency 'mail'
+  spec.add_dependency 'net-smtp'
```

Comment:
This gem is needed as Ruby 3.1 doesn't have 'net/smtp' package included.
https://github.com/mikel/mail/pull/1472

Or should we move this to regular `Gemfile` since we not require ActionMailer (thus are not using `mail` in runtime)?